addons/mail/controllers/thread.py

```diff
@@ -105,6 +105,8 @@ def mail_message_post(self, thread_model, thread_id, post_data, context=None, **
                 kwargs["partner_emails"], kwargs.get("partner_additional_values", {})
             )]
         post_data["partner_ids"] = list(set((post_data.get("partner_ids", [])) + new_partners))
+        if "everyone" in special_mentions:
+            post_data["partner_ids"] = [channel_member.partner_id.id for channel_member in thread.channel_member_ids]
```

Comment:
Not all members are partners: when they are not partners, they are `mail.guest`. Make sure to filter-out the guests.

_(it's fine if they are not notified at the moment, this feature is not available on / for Guests at the moment)_